src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Color[] COLOR = Color.values();
```

Comment:
enum name - singular
array name - plural
```suggestion
    private static final Color[] COLORS = Color.values();
```